test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,189 @@
+   
+   Use the following equation to configure how long an experiment should take in seconds:
+
+   $$t_{exp} = [stepDelay \times \frac{numberOfWorkloadObjects}{burstSize}] + \alpha$$
+
+   Where $\alpha <= 60$ seconds and it corresponds to the amount of time that clusterloader2 takes to cleanup resources after successful execution of a test. 
+   
+   For example, using the default parameters (i.e., burstSize=1, stepDelay=60 seconds, 10 workload objects) the experiment will take 660 seconds at most. Below are some sample parameters:
+
+   | burstSize (#)   | stepDelay (sec) |  numberOfWorkloadObjects  | duration |
+   | --------------- | --------------- | ------------------------- |--------- |
+   |       1         |      60         |           1,440           |  1 day   |
+   |       1         |      5          |          120,960          |  7 days  |
+
+
+4. Deploy your first workload:
+
+   First, use a kubeconfig that includes contexts for WDS spaces and set its current context to a target WDS space (e.g., `wds1`):
+
+   ```bash 
+   kubectl config use-context wds1
+   ```
+
+   Output:
+   ```console 
+   Switched to context "wds1".
+   ```
+   
+   Optionally, check the kubeconfig contexts: 
+   ```bash 
+   kubectl config get-contexts
+   ```
+   
+   Output:
+   ```console 
+   CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+             cluster1        kind-cluster1   kind-cluster1   
+             cluster2        kind-cluster2   kind-cluster2   
+             its1            its1-cluster    its1-admin      
+             kind-kubeflex   kind-kubeflex   kind-kubeflex   
+   *         wds1            wds1-cluster    wds1-admin      default
+             wds2            wds2-cluster    wds2-admin      default
+   ```
+
+   Second, create the binding Policy and namespace for your experiment using the following commands:
+
+   ```bash 
+   cd $CL2_DIR/testing/load/
+   kubectl apply -f long-duration-exp-setup.yaml
+   ```
+
+   Lastly, set the ``--kubeconfig`` flag to the path of the kubeconfig file used above and use the newly added Kubestellar provider (`--provider=ks`). Then, run the following command to create the workload:
+
+   ```bash
+   cd $CL2_DIR
+   go run cmd/clusterloader.go --testconfig=./testing/load/long-duration-exp-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
+   ```
+    
+   The generated workload is a pod that sleeps for 20 seconds([see here](workloads/long-duration-exp-workload.yaml)). Proceed to the next step below and execute it in parallel.
```

Comment:
I think that the reader needs some more explanation here. Is it _necessary_ to do step 5 in parallel, or merely _allowed_? If it is necessary, how promptly does the user have to proceed? Does anything get lost if the user does not act quickly enough (if so, what)?

Reply:
It's merely _allowed_ to do step 5 in parallel. Hence, I **removed** the sentence below to avoid any confusion to the reader: 

```
Proceed to the next step below and execute it in parallel.
```

Reply:
Even if doing step 5 in parallel with step 4 is not required, I suspect that it is recommended. In any case, I think this document should at least suggest it.

Reply:
ok, I see. I update with the following new statement: 

``The generated workload is a pod that sleeps for 20 seconds ([see here](https://github.com/dumb0002/kubestellar/blob/long-duration-exp/test/performance/long-running-tests/workloads/long-duration-exp-workload.yaml)). Proceed to the next step below and we recommend to execute it in parallel to step 4, if you want to mainting the minimal number of workload objects in your environment. Beware that any delay in the execusion of step 5 as it may lead to an accumulation of workload objects, potentially overloading the system if the delay continues for an extended period.``